examples/suit_update/README.md

```diff
@@ -483,10 +484,249 @@ success, the application reboots on the new slot:
+
+The manifest version indicates the SUIT manifest specification version numbers,
+this will always be 1 for now. The sequence number is the monotonically
+increasing anti-rollback counter.
+
+Each component, or payload, also has a number of parameters. The install-id
+indicates the unique path where this component must be installed.
+The vendor and class ID are used in manifest conditionals to ensure that the
+payload is valid for the device it is going to be installed in. It is generated
+based on the UUID(v5) of `riot-os.org` and the board name (`native`).
+
+The file and uri are used to generated the URL parameter and the digest in the
+manifest. The bootable flag specifies if the manifest generator should instruct
+the node to reboot after applying the update.
+
+Generating the actual SUIT manifest from this is done with:
+
+```console
+$ dist/tools/suit/suit-manifest-generator/bin/suit-tool create -f suit -i suit.tmp -o coaproot/suit_manifest
+```
+
+This generates the manifest in SUIT CBOR format. The content can be inspected by
+using the `parse` subcommand:
+
+```console
+$ dist/tools/suit/suit-manifest-generator/bin/suit-tool parse -m suit_manifest
+```
+
+The manifest generated doesn't have an authentication wrapper, it is unsigned
+and will not pass inspection on the device or RIOT instance. The manifest can be
+signed with the `sign` subcommand together with the keys generated earlier.
+
+```console
+$ dist/tools/suit/suit-manifest-generator/bin/suit-tool sign -k keys/default.pem -m coaproot/suit_manifest -o coaproot/suit_manifest.sign
+```
+
+This generates an authentication to the manifest. This is visible when
+inspecting with the `parse` subcommand. The URL to this signed manifest will be
+submitted to the instance so it can retrieve it and in turn retrieve the
+component payload specified by the manifest.
+
+#### Updating the storage location
+
+The update process is a two stage process where first the instance receives the
+URL for the manifest. It will download the manifest and verify the content.
+After the manifest is verified, it will proceed with executing the command
+sequences in the manifest and download the payload when instructed to.
+
+The URL for the manifest can be submitted to the host with any coap client. In
+this example the `coap-client` from libcoap is used:
+
+```console
+$ coap-client coap://[2001:db8::2]/suit/trigger -m post -e 'coap://[2001:db8::1]/suit_manifest.signed'
+```
+
+The payload is the full URL to the signed manifest. The native instance should
+respond on this by downloading and executing the manifest. If all went well, the
+output of the native instance should look something like this:
+
+```
+suit: received URL: "coap://[2001:db8::1]/suit_manifest.signed"
+suit_coap: trigger received
+suit_coap: downloading "coap://[2001:db8::1]/suit_manifest.signed"
+suit_coap: got manifest with size 278
+suit: verifying manifest signature
+suit: validated manifest version
+)Retrieved sequence number: 0
```

Comment:
Seems that the `storage_set_seq_no` is missing. For regular (flash-based) updates this is not an issue because the sequence number is part of the blob and flashed together with the firmware, but it is an issue when the storage backend tracks the sequence number separately from the blob.